app/src/renderer/vuex/modules/send.js

```diff
@@ -89,13 +89,23 @@ export default ({ node }) => {
       const submitType = args.submitType || `local`
       delete args.submitType
 
-      // extract "to" address
+      // TODO: refactor to get path request parameters at once
+      // extract path parameters
       const to = args.to
```

Comment:
would `toAddress` be more descriptive?

Reply:
will address that on the refactor PR for this, I'd like to get the path params for the endpoints at once